creme/creme_core/tests/forms/test_widgets.py

```diff
@@ -43,7 +44,7 @@ def test_render_no_l10n_01(self):
         name = 'test_calendar'
         value = self.formfield_value_date(2022, 2, 25)
         # help_text = _('Format: Day-Month-Year (Ex:31-12-2022)')
-        help_text = _('E.g. {}').format('2022-12-31')
+        help_text = _('E.g. {}').format(f'{date.today().year}-12-31')
```

Comment:
I have already fixed that in the branch "v2.4-rc2" (with hard coded "2023" value to avoid doing the same thing in code & unit test :thinking: )